src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -610,10 +610,12 @@ private Dataset parseDataset(String datasetJson) throws WrappedResponse {
     @GET
     @AuthRequired
     @Path("{identifier}")
-    public Response viewDataverse(@Context ContainerRequestContext crc, @PathParam("identifier") String idtf) {
+    public Response viewDataverse(@Context ContainerRequestContext crc, @PathParam("identifier") String idtf, @QueryParam("returnOwners") Boolean returnOwners) {
```

Comment:
To be consistent with method naming in other places, why not rename the endpoint method from "viewDataverse" to "getDataverse"?